MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -21,11 +21,16 @@ struct FarmerDetailsViewModel: Identifiable, Hashable {
     let farmerAddressesTypes: [String]
     let city: String
 
+    var getPhoneCallURL: URL {
+        guard let phoneNumber, let url = URL(string: "tel:\(phoneNumber)") else { return URL(fileURLWithPath: "") }
```

Comment:
You really want an empty URL? `URL(fileURLWithPath: "")`
I don't think so. Your view will try to open a wrong URL. That's a bad process. You should put your URL as optional and let the view handle this case: dont display the phone call button for example or make it disabled